Sources/Public/DotLottie/DotLottieFile.swift

```diff
@@ -0,0 +1,152 @@
+//
+// DotLottie.swift
+// LottieFiles
+//
+// Created by Evandro Harrison Hoffmann on 27/06/2020.
+// Copyright © 2020 LottieFiles. All rights reserved.
+//
+
+import Foundation
+
+// MARK: - DotLottieFile
+
+/// Detailed .lottie file structure
+public final class DotLottieFile {
+
+  // MARK: Lifecycle
+
+  /// Loads `DotLottie` from `URL` containing .lottie file.
+  ///
+  /// - Parameters:
+  ///  - url: URL to .lottie file
+  ///  - Returns: Deserialized `DotLottie`. Optional.
+  init(url: URL) throws {
```

Comment:
Can we make this `async` since it's doing some heavy file operations? We don't want this to be called from the main thread